internal/provider/providers/porkbun/api.go

```diff
@@ -172,15 +195,16 @@ func (p *Provider) updateRecord(ctx context.Context, client *http.Client,
 }
 
 // See https://porkbun.com/api/json/v3/documentation#DNS%20Delete%20Records%20by%20Domain,%20Subdomain%20and%20Type
-func (p *Provider) deleteAliasRecord(ctx context.Context, client *http.Client) (err error) {
+func (p *Provider) deleteRecord(ctx context.Context, client *http.Client, recordType string) (err error) {
 	var subdomain string
 	if p.owner != "@" {
-		subdomain = p.owner + "."
+		subdomain = p.owner
```

Comment:
That was a bug as well right? There shouldn't be a suffix dot in that case, correct?

Reply:
yep, it's its own commit. I'll split into seperate PRs as I mentioned above

Reply:
moved to #775